imap_processing/cdf/config/imap_idex_l1b_variable_attrs.yaml

```
@@ -0,0 +1,330 @@
+int_fillval: &int_fillval -9223372036854775808
+
+max_uint16: &max_uint12 4095
+
+data_min: &data_min 0
+data_max: &data_max 4096
+
+string_base_attrs: &string_base
+  CATDESC: " "
+  FIELDNAM: " "
+  FORMAT: A3
+  LABLAXIS: "none"
+  VAR_TYPE: metadata
+  DISPLAY_TYPE: no_plot
+  DEPEND_0: epoch
+  UNITS: " "
+
+l1b_data_base: &l1b_data_base
+  VALIDMIN: *data_min
+  VALIDMAX: *data_max
+  DISPLAY_TYPE: spectrogram
+  DEPEND_0: epoch
+  FORMAT: I12
+  UNITS: pC
+  VAR_TYPE: data
+  CATDESC: ""
+  FIELDNAM: ""
+  FILLVAL: *int_fillval
+  LABLAXIS: ""
+
+l1b_tof_base: &l1b_tof_base
+  <<: *l1b_data_base
+  DEPEND_1: time_high_sr
+
+l1b_target_base: &l1b_target_base
+  <<: *l1b_data_base
+  DEPEND_1: time_low_sr
+
+trigger_base: &trigger_base
+  VALIDMIN: 0
+  VALIDMAX: *max_uint12
+  DEPEND_0: epoch
+  DISPLAY_TYPE: no_plot
+  CATDESC: ""
+  FIELDNAM: ""
+  VAR_TYPE: support_data
+  FILLVAL: *int_fillval
+  LABLAXIS: ""
+  FORMAT: I10
+  UNITS: " "
+
+
+tof_high:
+  <<: *l1b_tof_base
+  CATDESC: Time of flight waveform on the high-gain channel
+  FIELDNAM: High Gain Time of Flight
+  LABLAXIS: TOF High Ampl.
```

remove period from all labelaxis?
```suggestion
  LABLAXIS: TOF High Ampl
```